tests/test_issues.cpp

```diff
@@ -177,7 +177,7 @@ void init_issues(py::module &m) {
     class MoveIssue1 {
     public:
         MoveIssue1(int v) : v{v} {}
-        MoveIssue1(const MoveIssue1 &c) { std::cerr << "copy ctor\n"; v=c.v; }
+        MoveIssue1(const MoveIssue1 &c) { v = c.v; }
```

Comment:
Whoops, good catch, that was a piece of debugging I accidentally left in the bug report that Wenzel copied as-is.  (Actually the non-debugging version was just ` = default;`, but that's basically just cosmetic.)